wear/src/main/AndroidManifest.xml

```diff
@@ -87,7 +87,16 @@
 
         <activity
             android:name=".conversation.ConversationActivity"
-            android:exported="true" />
+            android:icon="@mipmap/ic_assist_launcher"
+            android:roundIcon="@mipmap/ic_assist_launcher"
```

Comment:
As it is the same as the regular icon, [it can be removed](https://developer.android.com/develop/ui/views/launch/icon_design_adaptive#:~:text=define%20an%20icon%20drawable%20resource%20using%20the%20android%3AroundIcon%20attribute%E2%80%94but%20only%20if%20you%20require%20a%20different%20icon%20asset%20for%20circular%20masks)

Reply:
thank you!